16_wordle/main.go

```diff
@@ -29,19 +29,19 @@ func main() {
 	}
 
 	chosenWord := validWords.Random()
-	// debug
-	chosenWordStr := string(chosenWord)
-	chosenWordStrSlice := strings.Split(chosenWordStr, "")
-	// debug
-	fmt.Println("chosenWord:", string(chosenWordStr))
+	fmt.Println("chosenWord:", string(chosenWord))
 
 	for i := 0; ; i++ {
 		fmt.Printf("Enter 5 letter word\n>")
 		fmt.Scan(&input)
 
 		// check characters
-		fmt.Println(checkChar(chosenWord, input))
-		fmt.Println(chosenWordStrSlice)
+		inputOutput := bytes.Split(input, []byte(""))
+		fmt.Println(string(bytes.Join(inputOutput[:], []byte(" "))))
```

Comment:
```suggestion
		fmt.Println(string(input))
```

Reply:
Yes I get it, but I wanted to match the output from the README, where one emoji block is directly underneath the input character.
There's probably better ways to achieve that, but it's what I could think of. Had with multiple variables, lines which looked worse I think.

Reply:
Ah, I see, then I would suggest to move this into a function :)

And you can simplify with:

```suggestion
		inputOutput := bytes.Join(bytes.Split(input, []byte("")),[]byte(" "))
		fmt.Println(string(inputOutput))
```

Reply:
Done